doc/source/getting_started/docker.rst

```diff
@@ -0,0 +1,343 @@
+.. _pymapdl_docker:
+
+****************
+MAPDL and Docker
+****************
+
+You can run MAPDL within a container on any OS using Docker and
+connect to it via PyMAPDL.
+
+There are several situations in which it is advantageous to run MAPDL
+in a containerized environment such as Docker or Singularity:
+
+- Run in a consistent environment regardless of the host OS.
+- Portability and ease of install.
+- Large-scale cluster deployment using Kubernetes
+- Genuine app isolation through containerization
+
+In this case, you use your local Python installation to
+connect to this instance.
+
+.. graphviz::
+
+    digraph {
+      node [];
+      subgraph cluster_frontend {
+        label="*HOST*";
+        Python;
+      subgraph cluster_backend {
+        MAPDL;
+        label="*DOCKER*";
+        }
+      }
+      Python -> MAPDL
+      MAPDL -> Python
+    }
+
+Requirements
+============
+
+You need to have access to a Docker image with MAPDL in it.
+Visit :ref:`ref_make_container` for more information on how
+to create your own MAPDL docker image.
+
+Once you have created and uploaded your Docker image to a registry,
+you can start to pull and use the image in other devices.
+
+.. warning::
+
+   MAPDL Docker images are not allowed to be shared in
+   public or free-to-access repositories or registries.
+   Doing this you violate Ansys policy.
```

Comment:
```suggestion
   Doing so violates Ansys policy.
```
What policy? Is there something we can link to that defines the policy referenced?

Reply:
this part is work in progress... I need to check with Aaron.